testData/org/rust/lang/core/resolve/fixtures/use/cycle.rs

```diff
@@ -0,0 +1,6 @@
+use foo;
```

Comment:
Can you add small comment in code, that explains how this test work. This selfcycle is not obvius.